### PR TITLE
Fixes for 11640

### PR DESCRIPTION
**Pull request by RichieEscarez**
@bgrant0607 @brendandburns @thockin

### CODE REVIEW

File: _includes/nav_v1.0.html [hunk 1]
```diff
@@ -8,13 +8,8 @@
 					GETTING STARTED
 					 -->
 	<li class="navContainer">Getting Started
```
Comment (lavalamp) [inline _includes/nav_v1.0.html]: where is the /li for this?

**Comment by lavalamp** [comment]
please add back that end element, then LGTM

File: _includes/nav_v1.0.html (hunk 1 above)
Comment (RichieEscarez) [inline _includes/nav_v1.0.html]: done

Comment (bgrant0607) [inline _includes/nav_v1.0.html]: What this HTML generated by a tool? The formatting is very inconsistent.

**Comment by bgrant0607** [comment]
LGTM. Merging.

File: _includes/nav_v1.0.html (hunk 1 above)
Comment (RichieEscarez) [inline _includes/nav_v1.0.html]: Manually created and edited with Textmate. I need to find a tool that will format it reliably. Textmate at one point reordered and changed all the nesting. I plan to continue working on this file and will clean up the formatting soon. I also have a to-do to add comments to all the javascript.

Comment (lavalamp) [inline _includes/nav_v1.0.html]: Just to confirm this is what you intended-- it used to be a sibling, but now it's a child of the "Getting Started" li. Is that right?

**Pull request merged**